src/library_gl.js

```diff
@@ -1400,6 +1411,97 @@ var LibraryGL = {
     GLctx.bufferSubData(target, offset, HEAPU8.subarray(data, data+size));
   },
 
+  // Queries EXT
```

Comment:
Indeed, but they look different from the spec: `WebGLQuery` versus `WebGLTimerQueryEXT`.